lib/screens/tutorial_screen.dart

```diff
@@ -61,57 +63,65 @@ class _TutorialScreenState extends State<TutorialScreen> {
     myBottomDrawerController = BottomDrawerController();
     slides = kTutorialItems
         .map((item) => Column(
-              children: <Widget>[
-                SizedBox(height: 0.2.sh),
-                Padding(
-                    padding: EdgeInsets.symmetric(horizontal: 0.22.sw),
-                    child: Image.asset(
-                      item[kImageTutorial],
-                      height: 40,
-                      width: 40,
-                      fit: BoxFit.fill,
-                    )),
-                SizedBox(height: 0.15.sh),
-                Text(item[kHeaderTutorial], style: TextStyle(fontSize: isTablet ? 16.sp : 18.sp, fontWeight: FontWeight.w800, color: EaselAppTheme.kDartGrey), textAlign: TextAlign.center),
-                const SizedBox(height: 15),
-                SizedBox(width: 0.63.sw, child: Text(item[kDescriptionTutorial], style: TextStyle(color: Colors.black, fontSize: 13.sp, fontWeight: FontWeight.w400), textAlign: TextAlign.center)),
-              ],
-            ))
+      children: <Widget>[
+        SizedBox(height: 0.1.sh),
+        Padding(
+            padding: EdgeInsets.symmetric(horizontal: 0.22.sw),
+            child: Image.asset(
+              item[kImageTutorial],
+              height: 10.h,
+              width: 40,
+              fit: BoxFit.fill,
+            )),
+        SizedBox(height: 0.1.sh),
+        Text(item[kHeaderTutorial], style: TextStyle(fontSize: isTablet ? 16.sp : 18.sp, fontWeight: FontWeight.w800, color: EaselAppTheme.kDartGrey), textAlign: TextAlign.center),
+        const SizedBox(height: 15),
```

Comment:
Should SizedBox have .h?

Reply:
its .sh. Means 10 percent of the screen height.

Reply:
Same here, I was looking at line 78, I should have put the line number